qiskit/transpiler/transpiler.py

```diff
@@ -204,56 +204,42 @@ def transpile_dag(dag, basis_gates=None, coupling_map=None,
     if initial_layout is None:
         initial_layout = Layout.generate_trivial_layout(*dag.qregs.values())
 
-    if pass_manager:
-        # run the passes specified by the pass manager
-        # TODO return the property set too. See #1086
-        dag = pass_manager.run_passes(dag)
-    else:
+    if pass_manager is None:
         # default set of passes
-        # TODO: move each step here to a pass, and use a default passmanager below
-        name = dag.name
 
-        dag = Unroller(basis_gates).run(dag)
-        dag = BarrierBeforeFinalMeasurements().run(dag)
+        pass_manager = PassManager()
+        pass_manager.append(Unroller(basis_gates))
+        pass_manager.append(BarrierBeforeFinalMeasurements())
```

Comment:
This shouldn't be needed. We explicitly had a line for it because there was no pass manager. Now that there is, it will automatically run since the Swappers have this as 'required'.

Reply:
I don't think we need it anymore.

Reply:
fixed in df32f5f9